### PR TITLE
copy feedback

### PR DESCRIPTION
**Pull request by Zhifei-Li**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -401,6 +401,9 @@ <h3>SigLens v/s ClickHouse</h3>
           <div id="docker" role="tabpanel" class="tab-pane active">
             <div class="code position-relative"><span id="install-text">curl -L https://siglens.com/install.sh | sh</span>
               <i class="fa-regular fa-clone position-absolute copy-btn" id="copyBtn" data-toggle="tooltip" ></i>
+              <svg class="check-mark" id="check-mark" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
```
Comment (nbhavana) [inline index.html:404]: Add the check  svg file locally in assets.

File: index.html [hunk 2]
```diff
@@ -410,6 +413,9 @@ <h3>SigLens v/s ClickHouse</h3>
         <div class="code position-relative ingest-Cmd" id="generate-data">
           <span class="cmd-box" id="cmd-box">curl -Ls https://siglens.com/sendevents.sh | sh</span>
           <i class="fa-regular fa-clone position-absolute copy-btn" id="copyCmdBtn" data-toggle="tooltip"></i>
+          <svg class="check-mark-data" id="check-mark-data" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+            <polyline points="4 12 9 17 20 6" />
```
Comment (nbhavana) [inline index.html:417]: same here, we should use the svg file from assets

File: index.html (hunk 1 above)
Comment (Zhifei-Li) [reply index.html:404]: This svg is an animation. The  tag of html does not support animation inside SVG, so we can only embed SVG code into the HTML document.

File: index.html (hunk 2 above)
Comment (Zhifei-Li) [reply index.html:417]: save as above

File: index.html (hunk 1 above)
Comment (Zhifei-Li) [reply index.html:404]: sorry, missing words: The  tag of html does not support animation inside SVG, so we can only embed SVG code into the HTML document.

**Review by nbhavana — APPROVED** [review]
lgtm

**Pull request merged**